source/develop/release-management/features/network/copy-host-networking.html.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Copy Host Networking
+category: feature
+authors: Ales Musil
+feature_name: Copy Host Networking
+feature_modules: copy-host-networking
+feature_status: In Development
+---
+
+# Copy Host Networking
+
+## Summary
+
+Adding multiple hosts to oVirt via the UI can be rather tedious. The slow point is that the administrator has to go
+through every new host and do Setup Host Network manually. By utilizing Copy Host Networking, the administrator has
+to set only single host, for the rest the administrator can just specify the host to take the network configuration from. 
+
+
+### Owner
+
+- Feature Owner: Ales Musil (amusil)
+- E-mail: <amusil@redhat.com>
+
+
+### Benefit to oVirt
```

Comment:
You can delete this section. It repeats the proposed introduction.

Reply:
Done